### ISSUE TITLE
List Operations -> Columns wrong syntax

### ISSUE THREAD

**Issue by realtebo** [Laravel-Backpack/CRUD#5269]
Code showed HERE [https://backpackforlaravel.com/docs/6.x/crud-operation-list-entries] is now

```
CRUD::column([
  'name' => 'name', // The db column name
  'label' => "Tag Name", // Table column heading
  'type' => 'Text'
]);
```

But should be using `addColumn`, because passing an array to column result in 

> mb_strlen(): Argument Laravel-Backpack/docs#1 ($string) must be of type string, array given

**Comment by tabacitu** [comment]
@pxpm I have tested and confirmed the issue:
- `CRUD::field($array)` works
- `CRUD::column($array` does NOT work

When I try it, I also get `mb_strlen(): Argument #1 ($string) must be of type string, array given`
Error stack here: https://flareapp.io/share/W7z86d2m

**Comment by pxpm** [comment]
Thanks @realtebo for the report 🙏 Should be fixed in 6.1.6 that I tagged a few seconds ago.

Much appreciated your help reporting issues 🥇 

Cheers
